app/models/conference.rb

```diff
@@ -595,7 +595,16 @@ def notify_on_registration_dates_changed?
   end
 
   def registration_limit_exceeded?
-    registration_limit > 0 && registrations.count >= registration_limit
+    registration_limit > 0 && registrations.count + unregistered_speakers >= registration_limit
+  end
+
+  # Returns the number of speakers of new, unconfirmed and confirmed events that are not registered
```

Comment:
Let's fix this comment as well and call the function unregistered_confirmed_speakers